src/compiler/corePublic.ts

```diff
@@ -52,7 +52,7 @@ export type Comparer<T> = (a: T, b: T) => Comparison;
 
 /** @internal */
 export const enum Comparison {
-    LessThan    = -1,
-    EqualTo     = 0,
-    GreaterThan = 1
+    LessThan = -1,
+    EqualTo = 0,
+    GreaterThan = 1,

```

Comment:
This is another case of a previously columnar list, though not comments. This is another case where I'd sorta like it to look like gofmt, like: https://go.dev/play/p/5AMAfNwVFzA